test/extended/authentication/front_proxy.go

```diff
@@ -33,6 +33,13 @@ var _ = g.Describe("[sig-auth][Feature:Authentication] ", func() {
 			if *controlPlaneTopology == configv1.ExternalTopologyMode {
 				e2eskipper.Skipf("External clusters do not have an aggregator-client secret in the cluster. Because the control plane lives outside the cluster, the aggregator-client secret is not needed in the cluster.")
 			}
+
+			isMicroShift, err := exutil.IsMicroShiftCluster(oc.AdminKubeClient())
```

Comment:
Instead of skipping, you should figure out a way to retrieve the client cert and test the front proxy with it, still.

Reply:
I think this test depends on user apigroup, so does it make sense to annotate it with `[apigroup:user.openshift.io]` ?

Reply:
Added https://issues.redhat.com/browse/OCPBUGS-17431 link as a reference to enable this test later on.